packages/events-example/package.json

```diff
@@ -24,7 +25,7 @@
     "eslint": "^8.21.0",
     "eslint-config-oclif": "^4",
     "eslint-config-oclif-typescript": "^1.0.2",
-    "globby": "^13",
+    "globby": "^11",
```

Comment:
This was weird. I was getting an error with the ES6 import styles between oclif and globby. Had to downgrade to fix. [Known issue with globby.](https://github.com/sindresorhus/globby/issues/193)

Error:
```
(node:84782) [ERR_REQUIRE_ESM] Error Plugin: @theniledev/events-example [ERR_REQUIRE_ESM]: require() of ES Module /Users/sdelamater/work/nile-js/packages/events-example/node_modules/globby/index.js from /Users/sdelamater/work/nile-js/node_modules/@oclif/core/lib/config/plugin.js not supported.
Instead change the require of index.js in /Users/sdelamater/work/nile-js/node_modules/@oclif/core/lib/config/plugin.js to a dynamic import() which is available in all CommonJS modules.
module: @oclif/core@1.13.0
```